site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -346,13 +346,13 @@ Use the tab JavaScript plugin—include it individually or through the compiled
   </ul>
   <div class="tab-content" id="myTabContent">
     <div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab" tabindex="0">
-      <p><strong>This is some placeholder content the Home tab's associated content.</strong> Clicking another tab will toggle the visibility of this one for the next. The tab JavaScript swaps classes to control the content visibility and styling. You can use it with tabs, pills, and any other <code>.nav</code>-powered navigation. No1</p>
+      <p><strong>This is some placeholder content the Home tab's associated content.</strong> Clicking another tab will toggle the visibility of this one for the next. The tab JavaScript swaps classes to control the content visibility and styling. You can use it with tabs, pills, and any other <code>.nav</code>-powered navigation.</p>
```

Comment:
I can understand the thought, but I would prefer at least, to emphasize only the different words and not the whole sentence. 

_(As you rightly imagined, as I developed this, I couldn't focus on the differences)_

Reply:
Yep good point! I've emphasized only `<strong>Home tab's</strong>` instead of the whole sentence in d711e82fa3da936eb563c838e0f379acbb2000e7.

However I'm not sure if it wouldn't be better in English to emphasize only `<strong>Home</strong>` or `<strong>Home tab</strong>'s` in good English :shrug:

Reply:
I prefer it as you did it. @mdo  are you fine with it?